manifests/params.pp

```diff
@@ -38,8 +38,8 @@
   $framework_config = {
     'framework.server.name'     => $::fqdn,
     'framework.server.hostname' => $::fqdn,
-    'framework.server.port'     => '4440',
```

Comment:
I'm confused; I just changed them to the ssl defaults. They're still there, and they correspond with the now-default SSL. Do you mean if people haven't explicitly disabled SSL?

It's my understanding that a new major version would solve that.